nplab/instrument/camera/Andor/andor_sdk.py

```diff
@@ -476,11 +477,11 @@ def capture(self):
             int         number of images taken
             tuple       shape of the images taken
         """
-
+        self.Shutter = (1, self.keep_shutter_open, 1, 1)
```

Comment:
Tbf, I don't fully understand what each of the elements in the self.Andor.Shutter property do, so I don't know why we'd be changing property values every time we take an acquisition

Reply:
When in kinetic mode, if Shutter == (1,0,1,1) the shutter opens and closes between each spectrum which introduces vibrations, cuts off some of the signal and limits the time resolution of the measurement. When Shutter == (1,1,1,1) it's kept open. It's a handy option to have, Bart has been coding in this option manually before and after each capture.

Reply:
Fair enough, for some reason I thought the SDK could do that automatically. 
So then I think that it should be done in the Andor class not the AndorBase class (it's a very specific use case). And (perhaps more importantly), this functionality shouldn't be setting the rest of the Shutter parameters.

Each of the values in the tuple mean something (type, mode, closing time, opening time), and they should be preserved